flaky/flaky_pytest_plugin.py

```diff
@@ -76,16 +76,15 @@ def pytest_runtest_protocol(self, item, nextitem):
                 self.min_passes,
             )
         original_call_runtest_hook = self.runner.call_runtest_hook
-        self._call_infos = {}
+        self._call_infos[item] = {}
```

Comment:
In the previous version of the code, `_call_infos` was an instance variable (because `self._call_infos = {}` would shadow the class variable). Now, you'll always be setting state on the class variable. Is this what you want? If not, you should have an `__init__` that does ``self._call_infos = {}``.